src/api/client/Assets.ts

```diff
@@ -173,8 +173,10 @@ export class Assets {
     const rawAssetIds = await asset.tickerAssetId.multi(rawTickers);
 
     return rawAssetIds.reduce<TickerReservation[]>((result, rawAssetId, index) => {
-      if (rawAssetId.isNone) {
-        const ticker = tickerToString(rawTickers[index]);
+      const rawTicker = rawTickers[index];
+
+      if (rawAssetId.isNone && rawTicker) {
+        const ticker = tickerToString(rawTicker);
```

Comment:
The was no need to add the extra condition here. `rawTickers[index]` will always be defined. A mere type assertion would have done the trick. 
```suggestion
      if (rawAssetId.isNone) {
        const ticker = tickerToString(rawTickers[index] as PolymeshPrimitivesTicker);
```